tests/integration/test_archives.py

```diff
@@ -64,7 +64,10 @@ def test_archives_put(
     cursor = dev_db_connection.cursor()
     cursor.execute(
         """
-    SELECT last_cached, broken_source_url_as_of FROM data_sources where airtable_uid = %s
+    SELECT last_cached, broken_source_url_as_of 
+    FROM data_sources 
+    INNER JOIN data_sources_archive_info ON data_sources.airtable_uid = data_sources_archive_info.airtable_uid 
+    WHERE data_sources.airtable_uid = %s
     """,
```

Comment:
Converting to a more vertical orientation here is much appreciated.

Reply:
Yeah it was running off the screen from all the things I added haha